src/index.js

```diff
@@ -1,3 +1,5 @@
+if (!global._babelPolyfill) { require('babel-polyfill'); }
```

Comment:
Not really anymore. It was more so that I could run tests that don't require feathers individually without having to run the whole suite.